specification/sql/resource-manager/Microsoft.Sql/preview/2015-05-01-preview/examples/ManagedInstanceList.json

```diff
@@ -22,7 +22,8 @@
               "state": "Ready",
               "vCores": 8,
               "storageSizeInGB": 1024,
-              "licenseType": "Full"
+              "licenseType": "Full",
+              "collation":  "SQL_Latin1_General_CP1_CI_AS"
```

Comment:
Looks like dnsZone is missing from all these other examples

Reply:
It has been added now